deploy/helm/sumologic/conf/metrics/metrics.conf

```diff
@@ -12,6 +12,8 @@
 <label @DATAPOINT>
   <filter prometheus.metrics**>
     @type enhance_k8s_metadata
+    cache_size  {{ .Values.sumologic.k8sMetadataFilter.bearerCacheSize }}
+    cache_ttl  {{ .Values.sumologic.k8sMetadataFilter.bearerCacheTtl }}
```

Comment:
should we also add these when we create a filter of type `enhance_k8s_metadata` in `logs.source.containers.conf`?

Reply:
good point. Will add it in the logs pipeline as well.